src/common/m_checker_common.fpp

```diff
@@ -138,6 +138,15 @@ contains
         @:PROHIBIT(hypoelasticity .and. model_eqns /= 2)
     end subroutine s_check_inputs_hypoelasticity
 
+    !> Checks constraints on the hyperelasticity parameters.
+        !! Called by s_check_inputs_common for pre-processing and simulation
+    subroutine s_check_inputs_hyperelasticity
+        if (model_eqns == 1 .or. model_eqns > 3) then
+            call s_mpi_abort('hyperelasticity requires '// &
+                             '6-equation model (model_eqns = 2 or 3). Exiting ...')
```

Comment:
This error message is unclear. It says hyperelasticity requires the 6-equation model, but then says you can pick from model_eqns = 2 or 3, which implies it can be used with the 5- and 6-equation models. Which is it?